user/code-climate.md

```diff
@@ -104,22 +114,6 @@ You can find additional setup, installation, and troubleshooting information her
 
 ## Common Problems
 
-#### I get an error transmitting coverage results
```

Comment:
Of course. We used to run our reporting within the test context, which meant that [WebMock](https://github.com/bblimke/webmock) users would inadvertently prevent us from making the network requests that submits the coverage data to our services. The workaround was to configure WebMock to allow network requests to our domain.

Now that we submit test coverage in a separate step, WebMock won't interfere with our network requests.